src/metabase/api/util.clj

```diff
@@ -47,12 +47,14 @@
   "URL for where to send product feedback"
   :type :string
   :default "https://store-api.metabase.com/api/v1/crm/product-feedback"
-  :visibility :internal)
+  :visibility :internal
+  :export? false)
```

Comment:
Linter was complaining about this, I set it to false as I don't think this will actually be changed by any real user

Reply:
Can you tell me your thinking behind this custom getter?

Reply:
I don't want people to start metabase with `yarn dev` and accidentally send feedback to the real endpoint when playing around with it.
I know we can still customize it with the env, but without the custom getter it would still default to the prod endpoint

Reply:
Not sure if the setting is still worth it as the setter is basically what could go in a (def var ...)

Reply:
ah i see!. So we don't want to invite anyone to change this, and we want to make sure that at dev time it is intentional. I think this might be better then:

```clojure
(defn- product-feedback-url
  "Product feedback url. When not prod, reads `MB_PRODUCT_FEEDBACK_URL` from the environment to prevent development
  feedback from hitting the endpoint."
  []
  (if config/is-prod?
    "https://store-api.metabase.com/api/v1/crm/product-feedback"
    (env :mb-product-feedback-url)))

(mu/defn send-feedback!
  "Sends the feedback to the api endpoint"
  [comments :- [:maybe ms/NonBlankString]
   source :- ms/NonBlankString
   email :- [:maybe ms/NonBlankString]]
  (try (http/post (product-feedback-url)
                  {:content-type :json
                   :body         (json/generate-string {:comments comments
                                                        :source   source
                                                        :email    email})})
       (catch Exception e
         (log/warn e)
         (throw e))))
```

this changes the `defsetting` to be a function. As a setting, it seems to invite people to change it, and what we really want to do is always use one during production and require a dev to set a value at dev time.

Does this seem right?

Reply:
That's perfect thanks 🙏 , pushed it in https://github.com/metabase/metabase/pull/40974/commits/2c27d07800b52df757fa137c8b442c3948bd8df5